src/VisualStudio/TestUtilities/Proxy/VisualStudioApplication.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Reflection;
+using Roslyn.VisualStudio.Test.Utilities.Remoting;
+
+namespace Roslyn.VisualStudio.Test.Utilities.Proxy
+{
+    internal class VisualStudioApplication
+    {
+        public static void WaitForApplicationIdle(VisualStudioInstance host)
```

Comment:
If anything, this method should be a public member of the `VisualStudioInstance` class.